biogeochem/FatesSoilBGCFluxMod.F90

```diff
@@ -31,6 +31,7 @@ module FatesSoilBGCFluxMod
   use PRTGenericMod     , only : repro_organ
   use PRTGenericMod     , only : struct_organ
   use PRTGenericMod     , only : SetState
+  use PRTAllometricCNPMod,only : stoich_max
```

Comment:
I think this needs to be made public in PRTAllometricCNPMod.F90:
https://github.com/NGEET/fates/blob/c8778be187b120362e35c539164f436c7701da74/parteh/PRTAllometricCNPMod.F90#L99

Manually updating my local branch of this for testing corrects the build error I got.

Reply:
resolved